tests/Integration/Behaviour/Features/Scenario/Product/update_prices_without_states.feature

```diff
@@ -0,0 +1,280 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-prices-without-states
+@restore-all-tables-before-feature
+@reboot-kernel-before-feature
+@clear-cache-before-feature
+@update-product-prices
+@update-prices-without-states
+@reboot-kernel-after-feature
+@clear-cache-after-feature
+Feature: Update product price fields from Back Office (BO).
+  As a BO user I want to be able to update product fields associated with price.
+
+  Background:
+    Given shop configuration for default shop fr
```

Comment:
sounds broken. Did you mean at least smth like?

```suggestion
    Given shop configuration for default shop is set to fr
```

Reply:
And i think this is the main and only difference from all the tests in `update_prices.feature` - the different country. As far as I remember you just expect to check different tax rate depending if BO country has state or not. So to reflect the actual scenario/usecase (and save some tests running time) it was probably enough just to add a single scenario into existing `update_prices.feature` ? smth like `Scenario: I update product prices when default country has no states`. Most of the times more tests = better, but in this case I think it actually hides the usecase and wastes the tests time, though I may be wrong :man_shrugging:  wdyt @jolelievre ?

Reply:
I can remove some test but i think because of cache its better to have it in different file. It fails in the same file and i think it would be difficult to clear all cache in middle of file. To avoid not needed difficulties i prefer to have it in new file but yeah maybe not all test are needed. What do you think @jolelievre @zuk3975